website/content/docs/operations/aws-oidc-provider.mdx

```diff
@@ -1,29 +1,51 @@
 ---
 layout: docs
-page_title: Nomad OpenID Connect with AWS Identity Provider
+page_title: Authenticate Nomad jobs with AWS IAM Identity Provider
```

Comment:
```suggestion
page_title: Authenticate Nomad jobs with AWS IAM identity as an OIDC provider
```
Did a bunch more reading and found that AWS IAM Identity Provider isn't actually a product. Also Nomad is the provider here so wanted to clarify that